oddt/toolkits/rdk.py

```diff
@@ -1252,11 +1279,14 @@ def bonds(self):
 
     @property
     def partialcharge(self):
+        if self.Atom.HasProp('_MMFF94Charge'):
+            return self.Atom.GetDoubleProp('_MMFF94Charge')
+        if self.Atom.HasProp('_GasteigerCharge'):
```

Comment:
Does it work for hydrogens? In `PDBQTAtomLines` you also check `'_GasteigerHCharge'`

Reply:
This should not happen in ODDT - `mol.addh()` purges computed props - but user can supply the molecule with such props.